post_office/templatetags/html_email.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do you mind renaming this file to `post_office.py`? This way post office tags can be loaded with `{% load post_office %}`